README.md

```diff
@@ -784,7 +784,10 @@ parameters:
       type: integer
     -
       name: direction
-      name: string
+      type: string
+      enumerate:
```

Comment:
See https://frictionlessdata.io/specs/table-schema/#constraints for correct syntax according to tableschema spec